tests/app/v2/notifications/test_post_letter_notifications.py

```diff
@@ -467,7 +521,7 @@ def test_post_letter_notification_persists_notification_reply_to_text(
 
     notifications = Notification.query.all()
     assert len(notifications) == 1
-    assert notifications[0].reply_to_text == service_address
```

Comment:
It might be the git diff but are we dropping a test here? The one that checks that the default service address is used if we don't override it at the create_template level?

Reply:
The expected behaviour is that the default service address _shouldn’t_ be used if `template.service_letter_contact_id` is set to `None`. 

That should be treated as the letter contact block being blank instead:
![image](https://user-images.githubusercontent.com/355079/78654247-2e0cd880-78bc-11ea-8cd5-806d953e4d66.png)